### PR TITLE
API1800 support for Logical Interconnects

### PR DESCRIPTION
**Pull request by yuvirani**
### Description
API1800 support for Logical Interconnects

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
  - [ ] New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
- [x] Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,9 +7,10 @@
 - oneview_ethernet_network
 - oneview_fc_network
 - oneview_fcoe_network
+- oneview_logical_interconnect
 - oneview_network_set
 - oneview_scope
-- oneview_server_hardware
+- oneview_server_hardwar
```
Comment (VenkateshRavula) [inline CHANGELOG.md]: correct the spelling

File: endpoints-support.md [hunk 2]
```diff
@@ -100,34 +100,36 @@
...
+|<sub>/rest/logical-interconnects/{id}/firmware</sub>                                     | PUT       | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/logical-interconnects/{id}/forwarding-information-base</sub>                  | GET      | :heavy_multiplication_x:       | :heavy_multiplication_x:       | :heavy_multiplication_x: | :heavy_multiplication_x: |:heavy_multiplication_x: |:heavy_multiplication_x: |:heavy_multiplication_x: | :heavy_multiplication_x:   | :heavy_multiplication_x:   |
 |<sub>/rest/logical-interconnects/{id}/forwarding-information-base</sub>                  | POST     | :heavy_multiplication_x:       | :heavy_multiplication_x:       | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x:   |
```
Comment (VenkateshRavula) [inline endpoints-support.md]: you missed to update these endpoints

File: endpoints-support.md [hunk 3]
```diff
@@ -100,34 +100,36 @@
...
+|<sub>/rest/logical-interconnects/{id}/snmp-configuration</sub>                           | GET      | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/logical-interconnects/{id}/snmp-configuration</sub>                           | PUT      | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |:white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/logical-interconnects/{id}/support-dumps</sub>                                | POST     | :heavy_multiplication_x:       | :heavy_multiplication_x: | :heavy_multiplication_x:  | :heavy_multiplication_x:  | :heavy_multiplication_x:  | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x: |
```
Comment (VenkateshRavula) [inline endpoints-support.md]: update this endpoint also

File: examples/logical_interconnect_group_synergy.rb [hunk 4]
```diff
@@ -1,120 +0,0 @@
-# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
```
Comment (VenkateshRavula) [inline examples/logical_interconnect_group_synergy.rb]: restore this file

File: CHANGELOG.md (hunk 1 above)
Comment (yuvirani) [reply CHANGELOG.md]: Done

File: endpoints-support.md (hunk 2 above)
Comment (yuvirani) [reply endpoints-support.md]: Done

File: endpoints-support.md (hunk 3 above)
Comment (yuvirani) [reply endpoints-support.md]: Done

File: examples/logical_interconnect_group_synergy.rb (hunk 4 above)
Comment (yuvirani) [reply examples/logical_interconnect_group_synergy.rb]: Done

**Review by VenkateshRavula — APPROVED** [review]

**Review by nabhajit-ray — APPROVED** [review]

**Pull request merged**